store/views.py

```diff
@@ -77,6 +97,44 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    response_data={
+        'message':None,
+    }
+    book_id=request.POST['bid']
+    book=BookCopy.objects.get(pk=book_id)
```

Comment:
This may fail with invalid book ID given in POST request, and would lead to server error. Expected behavior is to inform user with Not found (404) error.